src/roulette.py

```diff
@@ -745,13 +745,12 @@ def placeBets(self) -> None:
         for more information.
         """
 
+    @abstractmethod
     def playing(self) -> bool:
```

Comment:
table minimum and rounds to go condition is a must for all players. better have it in base class.

Reply:
better not make this method abstract. child classes will override this only if have extra playing rules, otherwise session will invoke the base class playing by default.

Reply:
Done.